### ISSUE TITLE
wandb and tensor board failed to sync

### ISSUE THREAD

**Issue by kingsj0405** [xinntao/BasicSR#323]
Following code works well.
I'm not sure what is the problem...

```
import wandb
from torch.utils.tensorboard import SummaryWriter

wandb.init(sync_tensorboard=True)
tb_logger = SummaryWriter(log_dir='.')

tb_logger.add_scalar('test', 1)
tb_logger.add_scalar('test', 2)
tb_logger.add_scalar('test', 3)
```

My `wandb` version is `0.10.8`

**Comment by xinntao** [comment]
do your local tensorboard work well? 
And is there any error or warning information in the log file?

**Comment by kingsj0405** [comment]
Sorry for late reply.

Yes local tensorboard works well.

And system information(e.g. cpu utilization, gpu utilization) is synced well too.
But only the values of `loss`, `acc` and `images`(I added) are failed to sync.

**Comment by kingsj0405** [comment]
Exactly same with https://github.com/xinntao/BasicSR/issues/341
